packages/experimental/Shadow/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@fluentui-react-native/experimental-shadow",
+  "version": "0.0.1",
+  "description": "A cross-platform Shadow component to assist in applying the Fluent shadow ramp",
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "typings": "lib/index.d.ts",
+  "onPublish": {
+    "main": "lib-commonjs/index.js",
+    "module": "lib/index.js"
+  },
+  "scripts": {
+    "build": "fluentui-scripts build",
+    "just": "fluentui-scripts",
+    "clean": "fluentui-scripts clean",
+    "lint": "fluentui-scripts eslint",
+    "depcheck": "fluentui-scripts depcheck",
+    "test": "fluentui-scripts jest",
+    "update-snapshots": "fluentui-scripts jest -u",
+    "verify-api": "fluentui-scripts verify-api-extractor",
+    "update-api": "fluentui-scripts update-api-extractor",
+    "prettier": "fluentui-scripts prettier",
+    "prettier-fix": "fluentui-scripts prettier --fix true"
+  },
+  "dependencies": {
+    "@fluentui-react-native/framework": ">=0.5.16 <1.0.0",
+    "@fluentui-react-native/tokens": ">=0.9.12 <1.0.0",
+    "@fluentui-react-native/use-tokens": ">=0.1.2 <1.0.0"
+  },
+  "devDependencies": {
+    "@fluentui-react-native/test-tools": ">=0.1.1 <1.0.0",
+    "@types/react-native": "^0.63.0",
+    "@uifabricshared/build-native": "^0.1.1",
+    "@uifabricshared/eslint-config-rules": "^0.1.1",
+    "react": "16.13.1",
+    "react-native": "^0.63.4"
+  },
+  "peerDependencies": {
+    "react": ">=16.13.1",
+    "react-native": ">=0.63.4"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/microsoft/fluentui-react-native.git"
+  },
+  "keywords": [
+    "react-native",
+    "shadow",
+    "fluentui"
+  ],
```

Comment:
What does the `keywords` field do?

Reply:
I guess it adds some key terms for searching on npmjs?

https://docs.npmjs.com/cli/v6/configuring-npm/package-json#keywords

Reply:
yea npm init asked me for some keywords so I just picked a few

Reply:
See also the bottom of this page: https://www.npmjs.com/package/@fluentui/react